extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,221 @@
+Only two properties are introduced with this extension and combine to describe a single value; the percentage of light that is transmitted through the surface of the material. These properties work together with the existing properties of the material to define the way light is modified as it passes through the substance. 
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**transmissionFactor** | `number` | The base percentage of light that is transmitted through the surface.| No, Default: `0.0 ` |
+|**transmissionTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the transmission percentage of the surface, stored in the `R` channel. This will be multiplied by `transmissionFactor`. | No |
+
+### transmissionFactor 
+The amount of light that is transmitted by the surface rather than diffusely re-emitted. This is a percentage of all the light that penetrates a surface (i.e. isn’t specularly reflected) rather than a percentage of the total light that hits a surface. A value of 1.0 means that 100% of the light that penetrates the surface is transmitted through.
+
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>Yellow baseColor with transmissionFactor = 1.0.</em></figcaption>
+</figure>
+
+### transmissionTexture 
+The `R` channel of this texture defines the amount of light that is transmitted by the surface rather than diffusely re-emitted. A value of 1.0 in the red channel means that 100% of the light that penetrates the surface (i.e. isn’t specularly reflected) is transmitted through. The value is linear and is multiplied by the transmissionFactor to determine the total transmission value.
+
+<figure>
+  <img src="./figures/TransmissionTexture.png"/>
+<figcaption><em>Controlling transmission amount with transmissionTexture.</em></figcaption>
+</figure>
+
+## Tinting
+You may notice that the transparent materials shown above are tinted. i.e. they aren't actually transmitting 100% of the non-reflected light. This is because `KHR_materials_transmission` specifies that the `baseColor` of the material be used to model the absorption of light by the surface. This allows materials like stained glass to be easily represented with this extension.
+Absorption is usually defined as an amount of light at each frequency that is absorbed over a given distance through a medium (usually described by Beer’s Law). However, since this extension deals exclusively with infinitely thin surfaces, we can treat absorption as a constant. In fact, rather than absorbed light, we can talk about its inverse: transmitted light. The `baseColor` of the material serves this purpose as it already defines how the light that penetrates the surface is colored by the material. In this model, the transmitted light will be modulated by this color as it passes through. Note that because `baseColor` is defined as sRGB, it must be converted to linear before being used in this calculation.
```

Comment:
Actually can I propose a bit of a rewrite?  I'm concerned we've buried the lede here.

```suggestion
## Tinting
The `baseColor` of the material, as defined in the [Metallic-Roughness Material](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#metallic-roughness-material)
section of the glTF specification, defines the amount of light at each frequency
that is transmitted (not absorbed) by the material. Absorption is usually defined
as an amount of light at each frequency that is absorbed over a given distance
through a medium (usually described by Beer’s Law). However, since this
extension deals exclusively with infinitely thin surfaces, we can treat
absorption as a constant. Therefore, its inverse, `baseColor` transmission,
can also be constant.  This provides the color tinting effect in the materials
shown above.
```

Reply:
I like Ed's the rephrasing, but two nits — in the last two sentences could you replace the words "its" and "This" with the terms they refer to? I had a bit of trouble tracking the pronouns at the end.

Reply:
I rephrased this a bit. Maybe it's clearer now?